src/getHumanAge.test.js

```diff
@@ -1,10 +1,23 @@
 'use strict';
 
+const { getHumanAge } = require('./getHumanAge');
+
 describe('getHumanAge', () => {
-  const { getHumanAge } = require('./getHumanAge');
+  test('should convert cat and dog ages correctly', () => {
```

Comment:
This is not an informative and clear test name. I would recommend splitting all your expects to separate tests and describing them like this:
'should return 0 for cat and dog when they are 14 years old'